Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/storage/entity/TraceLocationConverters.kt

```diff
@@ -0,0 +1,13 @@
+package de.rki.coronawarnapp.eventregistration.storage.entity
+
+import androidx.room.TypeConverter
+import de.rki.coronawarnapp.eventregistration.events.TraceLocation
+
+class TraceLocationConverters {
+
+    @TypeConverter
+    fun toTraceLocationType(value: Int) = enumValues<TraceLocation.Type>()[value]
```

Comment:
`fromTraceLocationType` maps and stores, based on the `value` property of the enum class `TraceLocation.Type`, but `toTraceLocationType` here uses the order of the enums via `enumValues`, so if someone changes the order of the enums in `Type` :point_right:  :bomb: :fearful:

Reply:
I see there are tests for this, which should fail, but this still seems unintentional.

Reply:
Fixed in a new commit